lib/commands/general.js

```diff
@@ -42,6 +43,93 @@ commands.mobileIsToastVisible = async function (opts = {}) {
   });
 };
 
+commands.mobileOpenDrawer = async function (opts = {}) {
+  const {element, gravity} = assertRequiredOptions(opts, ['element']);
+
+  return await this.espresso.jwproxy.command(`/appium/execute_mobile/${util.unwrapElement(element)}/open_drawer`, 'POST', {
+    gravity
+  });
+};
+
+commands.mobileCloseDrawer = async function (opts = {}) {
+  const {element, gravity} = assertRequiredOptions(opts, ['element']);
+
+  return await this.espresso.jwproxy.command(`/appium/execute_mobile/${util.unwrapElement(element)}/close_drawer`, 'POST', {
+    gravity
+  });
+};
+
+commands.mobileSetDate = async function (opts = {}) {
+  const {element, year, monthOfYear, dayOfMonth} = assertRequiredOptions(opts, ['element']);
```

Comment:
are year, monthOfYear and dayOfMonth also required?

Reply:
Yep, fixed